docker/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Prepare the image with:
 #   docker build -t tensorflow/haskell:v0 docker
-FROM tensorflow/tensorflow:1.0.0
+FROM tensorflow/tensorflow:1.3.0
```

Comment:
Ah, I missed this when doing the update. Did using the old image cause any problems for you? It seems to work fine for me with 1.0.0 (but we should update it of course).